pkgs/development/web/nodejs/v18.nix

```diff
@@ -1,8 +1,20 @@
-{ callPackage, fetchpatch, openssl, python3, enableNpm ? true }:
+{ callPackage, lib, overrideCC, pkgs, buildPackages, fetchpatch, openssl, python3, enableNpm ? true }:
 
 let
+  # Clang 16+ cannot build Node v14 due to -Wenum-constexpr-conversion errors.
```

Comment:
Nit: did you mean Node v18 here, or did I misunderstand something?

Reply:
Copy-paste mistake. It’s the same solution as nodejs_14 copied into nodejs_18. I can push an update that corrects the comment.

Reply:
Thank you. An update wasn't necessary; I'm just an eager onlooker from the peanut gallery.